modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -99,36 +103,38 @@ output application/avro schemaUrl="classpath://schema.json"
 ----
 
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
-
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for Avro.
+DataWeave supports the following configuration properties for the Avro format.
 
-=== Reader Properties (for Avro)
+
+[[reader_properties]]
+=== Reader Properties
 
 The Avro format accepts properties that provide instructions for reading input data.
 
+
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `schemaUrl`   (Required) | `String` | `null` | The URL for the Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
+|Parameter |Type |Default|Description
+|`schemaUrl`  (Required) |`String`|`null`|The URL for the `Avro` schema. Valid values are `classpath://`, `file://`, or `http://`.
 |===
 
-=== Writer Properties (for Avro)
+[[writer_properties]]
+=== Writer Properties
 
 The Avro format accepts properties that provide instructions for writing output data.
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `bufferSize` | `Number` | `8192` | Size of the buffer writer.
-| `deferred` | `Boolean` | `false` | When set to `true`, DataWeave generates the output as a data stream, and the script's execution is deferred until it is consumed.
-  Valid values are `true` or `false`.
-| `schemaUrl`  (Required) | `String` | `null` | The URL for Avro schema. Valid values are `classpath://`, `file://`, or `http://`.
+|Parameter |Type |Default|Description
+|`bufferSize`  |`Number`|`8192`|Size of the buffer writer.
+|`deferred`  |`Boolean`|`false`|When set to `true`, DataWeave generates the output as a data stream, and the script's execution is deferred until it is consumed. Valid Options: `true` or `false`.
```

Comment:
TODO: "Valid Options:" everywhere -> "Valid values are" and needs to work after special formatting, like bullets, etc.